app/pages/dashboard/template.test.js

```diff
@@ -0,0 +1,54 @@
+import { createTestHarness } from '../../test-utils/testHarness';
+import context from './manifest.json';
+
+const setup = {
+  template: {
+    path: 'pages/dashboard/template.njk',
+  },
+};
+
+describe('organisations dashboard page', () => {
+  it('should render a backLink to the home page', createTestHarness(setup, (harness) => {
+    harness.request(context, ($) => {
+      const homepageBackLink = $('[data-test-id="go-back-link"]');
+      expect(homepageBackLink.length).toEqual(1);
+      expect(homepageBackLink.text().trim()).toEqual('Go back to previous page');
+      expect($(homepageBackLink).find('a').attr('href')).toEqual('/');
+    });
+  }));
+
+  it('should render a title', createTestHarness(setup, (harness) => {
+    harness.request(context, ($) => {
+      const title = $('h1[data-test-id="org-page-title"]');
+      expect(title.length).toEqual(1);
+      expect(title.text().trim()).toEqual(context.title);
+    });
+  }));
+
+  it('should render a description', createTestHarness(setup, (harness) => {
+    harness.request(context, ($) => {
+      const description = $('[data-test-id="org-page-description"]');
+      expect(description.length).toEqual(1);
+      expect(description.text().trim()).toEqual(context.description);
+    });
+  }));
+
+  it('should render a add organisation button', createTestHarness(setup, (harness) => {
+    harness.request(context, ($) => {
+      const addOrgButton = $('[data-test-id="add-org-button"] a');
+      expect(addOrgButton.length).toEqual(1);
+      expect(addOrgButton.text().trim()).toEqual('Add an organisation');
+      expect(addOrgButton.attr('href')).toEqual('#');
+    });
+  }));
+
+  it('should render a table', createTestHarness(setup, (harness) => {
```

Comment:
Can we be bit more specific on the name of this test. `Should render the registered organisations` perhaps

Reply:
well we are checking it just renders the table component?